src/dependabot/update_metadata.ts

```diff
@@ -4,20 +4,30 @@ export interface updatedDependency {
   dependencyName: string,
   dependencyType: string,
   updateType: string,
+  directory: string,
+  packageEcosystem: string,
+  targetBranch: string
 }
 
-export function parse (commitMessage: string): Array<updatedDependency> {
+export function parse (commitMessage: string, branchName: string, mainBranch: string): Array<updatedDependency> {
   const yamlFragment = commitMessage.match(/^-{3}\n(?<dependencies>[\S|\s]*?)\n^\.{3}\n/m)
 
-  if (yamlFragment?.groups) {
+  if (yamlFragment?.groups && branchName.startsWith('dependabot')) {
     const data = YAML.parse(yamlFragment.groups.dependencies)
 
+    const delim = branchName[10]
```

Comment:
let's make a note of why the delimiter is `branchName[10]` 😄 
```suggestion
    # Since we are on the `dependabot` branch (9 letters), the 10th letter in the branch name is the delimiter
    const delim = branchName[10]  
```